demo/index.html

```diff
@@ -89,10 +89,12 @@
                             <li><a href="demo-details.html">Demo details page</a></li>
                             <li><a href="https://github.com/ubuntudesign/maas-gui-vanilla-theme">MAAS GUI GitHub Project</a></li>
                         </ul>
-                        <ul class="u-float--right">
-                            <li><a href="http://maas.io">MAAS Website</a></li>
-                            <li><a href="https://github.com/ubuntudesign/vanilla-framework">Vanilla Framework</a></li>
-                        </ul>
+                        <div class="u-float--right">
+                            <ul class="u-float--right">
```

Comment:
What's the reasoning behind the double float right here? To keep it consistent with when there are notifications?